tests/lametro/test_bills.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from pupa.scrape.bill import Bill
+
+    
+def test_is_restricted(scraper, matter, bill_data_updates):
+    '''
+    Tests that `_is_restricted` returns the correct value, given
+    a dict of matter values.
+    '''
+    field,value,assertion = bill_data_updates
+    matter[field] = value
+
+    assert scraper._is_restricted(matter) == assertion
+
+
+def test_scraper(scraper, matter, bill_data_updates, mocker):
+    '''
+    Test that the scraper correctly assigns the value of 'restrict_view'
+    to bill extras.
+    '''
+    field,value,assertion = bill_data_updates
+    matter[field] = value
+    mocker.patch('lametro.LametroBillScraper.matter', return_value=matter)
+
+    for bill in scraper.scrape(matter_ids='2017-0643'):
```

Comment:
it looks like the matter intro date on the canned bill is always after the cutoff, so we aren't testing whether we aren't scraping bills before that date; and absent that, this test effectively tests the same functionality as `test_is_restricted`. am i missing a way they are different?

Reply:
`test_is_restricted` test a single function within the scraper class, and `test_scraper` tests the scrape functionality (which, yes, relies upon the efficacy of `test_is_restricted`). I think having both makes sense. Still, we do need something to test whether or not the scraper skips early bills! Change forthcoming.